e2e-tests/functions

```diff
@@ -703,3 +704,20 @@ wait_cluster_consistency() {
 		wait_sts_rollout "$st"
 	done
 }
+
+deploy_cert_manager() {
+	echo 'deploy cert manager'
+	kubectl create namespace cert-manager || :
+	kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true || :
+	kubectl apply -f "https://github.com/jetstack/cert-manager/releases/download/v${CERT_MANAGER_VER}/cert-manager.yaml" --validate=false || : 2>/dev/null
+	if [ "$OPENSHIFT" == "4.10" ]; then
```

Comment:
I think we do not support OPENSHIFT 4.10 long time ago

Reply:
https://github.com/percona/percona-postgresql-operator/pull/876/commits/ed581e5e7f58eec78e7b9ae6bb38a70e22fe8510